packages/builders/src/interactions/contextMenuCommands/Assertions.ts

```diff
@@ -31,17 +33,19 @@ export function validateRequiredParameters(name: string, type: number) {
 	validateType(type);
 }
 
-const dmPermissionPredicate = s.boolean.nullish;
+const dmPermissionPredicate = s.boolean.nullish.setValidationEnabled(isValidationEnabled);
 
 export function validateDMPermission(value: unknown): asserts value is boolean | null | undefined {
 	dmPermissionPredicate.parse(value);
 }
 
-const memberPermissionPredicate = s.union(
-	s.bigint.transform((value) => value.toString()),
-	s.number.safeInt.transform((value) => value.toString()),
-	s.string.regex(/^\d+$/),
-).nullish;
+const memberPermissionPredicate = s
+	.union(
+		s.bigint.transform((value) => value.toString()),
+		s.number.safeInt.transform((value) => value.toString()),
+		s.string.regex(/^\d+$/),
+	)
+	.nullish.setValidationEnabled(isValidationEnabled);
```

Comment:
Let's not disable validation here, as the constraints also handle transforming the input to valid bitfields (BitInts cannot be JSON stringified, floats are not allowed)